src/server/worker/api_server.go

```diff
@@ -111,6 +115,8 @@ type APIServer struct {
 	pipelines col.Collection
 	// The progress collection
 	plans col.Collection
+	// The shards collection
```

Comment:
I think this would be a good place to describe a bit more what kind of data is stored in this collection, unlike `jobs` and `pipelines`, `shards` is a lot less self explanatory.
Also while you're here would you mind updating the comment above on `plans`? It could use a little more explanation too, and definitely should say "The progress collection," I think that description is the product of bad autocomplete on my part.

Reply:
Sounds good.